mediation_service/tests/test_prepare_ssp_response.py

```diff
@@ -49,6 +54,191 @@ def test_remove_non_allergy_intolerance():
     assert expected_res == act
 
 
+def test_return_active_and_resolved_allergy():
+    # Given
+    fhir_res = {
+        "entry": [
```

Comment:
Really not a fan of these big data comparisons for testing, huge swathes of text like this tend to make it much harder to read the testing code and are an absolute nightmare to uplift. I'd rather test the unit functionality wherever possible in place of these integrated tests, but in place of re-writing all these tests could we at least move this expected/input data out into a `test_data` file or something for improved readibility?